chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -154,34 +154,16 @@ Zotero_Preferences.Export = {
 		var menulist = document.getElementById('zotero-quickCopy-locale-menu');
 		var menulistLabel = document.getElementById('zotero-quickCopy-locale-menu-label');
 		
-		var lastSelectedLocale = menulist.value;
-		var localeLabel = "";
-		if (menulist.disabled === true) {
-			menulist.removeItemAt(0);
-		}
-		
 		if (mode == 'bibliography') {
-			var defaultStyleLocale = Zotero.Styles.get(format).locale;
-			if (defaultStyleLocale) {
-				if (Zotero.Styles.locales[defaultStyleLocale] !== undefined) {
-					localeLabel = Zotero.Styles.locales[defaultStyleLocale];
-				} else {
-					localeLabel = defaultStyleLocale;
-				}
-			}
-		}
-		
-		if (mode == 'bibliography' && !defaultStyleLocale) {
-			menulist.value = lastSelectedLocale;
-			menulist.disabled = false;
+			let style = Zotero.Styles.get(format);
+			menulist.value = style.locale || this._lastSelectedLocale;
+			
 			menulistLabel.disabled = false;
+			menulist.disabled = !!style.locale;
 		} else {
-			menulist.insertItemAt(0, localeLabel, lastSelectedLocale);
-			menulist.selectedIndex = 0;
+			menulist.value = '';
```

Comment:
This would set the extensions.zotero.export.quickCopy.locale pref to "". In my original code, this pref remains untouched when selecting an export translator.